frontend/src/components/AccountReceiveTable.vue

```diff
@@ -60,12 +60,45 @@
         >.
```

Comment:
I don't think you need to sign to withdraw funds?
<img width="509" alt="image" src="https://github.com/ScopeLift/umbra-protocol/assets/61768337/e7d6502c-64d3-4914-a48d-e249083075e1">

Reply:
I see why you added that. But would it be too much trouble to change it to asking people to sign when they lick on withdraw instead.

Reply:
cool will update

Reply:
https://github.com/ScopeLift/umbra-protocol/pull/685/commits/29f6f6ecf25f892f33e0f4e8d027852ecf78c72a

updated the wallet store to have a `needSignature` computed val to be able to use freely throughout...intentionally used that value only in the relevant parts for this pr. used this value to update the label and disabled props on the withdraw form button.

Reply:
I think it would be better if we made the `Need Signature` button a `Sign` button if we don't have user signature, and then change it to `Withdraw` like it used to be once the user signs.

Reply:
updated:

https://github.com/ScopeLift/umbra-protocol/pull/685/commits/a211d4623617bfc752678eda2025280753219958

Reply:
Still some issues: 

https://github.com/ScopeLift/umbra-protocol/assets/61768337/75f79027-83b3-4249-949d-e6984b0571f6

Reply:
https://github.com/ScopeLift/umbra-protocol/pull/685/commits/d8b730b65a142fcd8beb5712d29c081a3c546a2b

Reply:
one last nit, let's rename the button name to "Withdraw" .

Reply:
or maybe "Sign and withdraw"

Reply:
https://github.com/ScopeLift/umbra-protocol/pull/685/commits/5d63f2a6a20968be854e2b8e2b58cc2bef5be880